mscore/editstyle.ui

```diff
@@ -247,7 +252,7 @@
        </size>
       </property>
       <property name="currentIndex">
-       <number>4</number>
+       <number>5</number>
```

Comment:
not needed (should rather be 1), but gets ignored anyhow and set programatically